components/builder-db/src/schema/project.rs

```diff
@@ -11,7 +12,7 @@ table! {
         owner_id -> BigInt,
         created_at -> Nullable<Timestamptz>,
         updated_at -> Nullable<Timestamptz>,
-        visibility -> Text,
+        visibility -> PackageVisibilityMapping,
```

Comment:
I don't see a migration for this? I suspect it will fail?

Reply:
This is copied over from packages, but I'm not seeing where it's defined either

Reply:
The DbEnum derives it. If you don't change the column type in the database to the `origin_package_visibility` enum, you're going to have a bad time

Reply:
Right, so that's in the migration
```
ALTER TABLE origin_projects ALTER COLUMN visibility SET DEFAULT 'public'::origin_package_visibility;
```

Reply:
example here: https://github.com/habitat-sh/builder/blob/master/components/builder-originsrv/src/migrations/2018-10-10-198563_enum_visibility/up.sql#L5-L10

Reply:
Ah ok, will update.. it definitely works fine without the type change.. it leaves the underlying type as text.